lib/drivers/docs/tandem.md

```diff
@@ -155,18 +162,18 @@ Settings history:
 
 No Tidepool data model yet:
 
-  - `[ ]` low insulin alert threshold
+  - `[-]` low insulin alert threshold
   - auto-off:
-    - `[ ]` enabled
-    - `[ ]` threshold
+    - `[-]` enabled
+    - `[-]` threshold
   - `[ ]` language
   - reminders:
-    - `[ ]` BG reminder
-    - `[ ]` bolus reminder
-  - `[ ]` alert settings (volume or vibration-only; whether enabled)
+    - `[-]` BG reminder
+    - `[-]` bolus reminder
+  - `[-]` alert settings (volume or vibration-only; whether enabled)
   - basal features:
     - `[ ]` temp basal type (`manual` or `percentage`)
-    - `[ ]` max basal (as a u/hr rate)
+    - `[-]` max basal (as a u/hr rate)
   - bolus features:
     - `[ ]` bolus "wizard"/calculator enabled
     - `[ ]` bolus increment for non-"quick"/manual boluses
```

Comment:
I think the bolus increment is available. Double check? (I can set it in the UI under Options -> My Pump -> Pump Settings; it's not part of the IDP though.)